plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp

```diff
@@ -205,7 +205,7 @@ class read_only {
 
    struct get_code_params {
       name account_name;
-      bool code_as_wasm = false;
+      bool code_as_wasm = true;
```

Comment:
Can you please make this code change in a separate commit? The current commit also updates the swagger file. We want to keep code changes separate from docs changes to separate concerns and simplify eventual git operations if needed. @b1bart is this change `false` -> `true` ok?

Reply:
done.